cassandra-driver/src/test/java/brave/cassandra/driver/ITTracingSession.java

```diff
@@ -209,8 +206,11 @@ void invokeBoundStatement() {
     session.execute(prepared.bind());
   }
 
-  static TraceContext extractB3Header() {
-    ByteBuffer b3 = CustomPayloadCaptor.ref.get().get("b3");
-    return B3SingleFormat.parseB3SingleFormat(UTF_8.decode(b3)).context();
+  @Nullable TraceContext extractB3Header() {
+    ArgumentCaptor<Statement> captor = ArgumentCaptor.forClass(Statement.class);
```

Comment:
this is neat for those wondering how to intercept something wrapped and look at its args